extensions/wikia/ImageReview/modules/PromoteImage/PromoteImageReviewHelper.class.php

```diff
@@ -117,15 +117,15 @@ protected function saveStats($statsInsert, $sqlWhere, $action) {
 				case '':
 					//	$stats['reviewer'] += count($images);
 					//	$stats['unreviewed'] -= count($images);
-					$stats['questionable'] += count($sqlWhere[ImageReviewStatuses::STATE_QUESTIONABLE]);
+					$stats['questionable'] += count($sqlWhere[ImageReviewStates::QUESTIONABLE]);
 					break;
 				case ImageReviewSpecialController::ACTION_QUESTIONABLE:
-					$changedState = count($sqlWhere[ImageReviewStatuses::STATE_APPROVED]) + count($sqlWhere[ImageReviewStatuses::STATE_REJECTED]);
+					$changedState = count($sqlWhere[ImageReviewStates::APPROVED]) + count($sqlWhere[ImageReviewStates::REJECTED]);
 					//	$stats['reviewer'] += $changedState;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)